tests/utils.go

```diff
@@ -2102,6 +2107,24 @@ func GetGuestAgentUserData() string {
                 `, ipv6UserDataString, GetUrl(GuestAgentHttpUrl), GetUrl(StressHttpUrl))
 }
 
+func GetIPv6NetworkData(ipAddress string, gateway string, dnsServer string, searchDomains []string) string {
```

Comment:
I would use the prefix `Generate` or `Create` and not `Get`. Dropping any prefix here will also work nicely.

I would also generalize it, accepting a list of interfaces where each has all the needed data, not limited to IPv6.
But this can be also done in the future.

Reply:
> I would use the prefix `Generate` or `Create` and not `Get`. Dropping any prefix here will also work nicely.

I see what you mean, but prefixing it `Get` is in line with the remaining functions - e.g. `GetGuestAgentUserData` for instance. I'll do it if you insist.

> 
> I would also generalize it, accepting a list of interfaces where each has all the needed data, not limited to IPv6.
> But this can be also done in the future.

+1 on doing it on a future PR.